cli/src/app.rs

```diff
@@ -119,5 +120,18 @@ pub fn app<'a>() -> clap::Command<'a> {
         .subcommand(
             Command::new("version")
                 .about("Display application version")
-        )
+        );
+
+    #[cfg(feature = "selfmanage")]
+    {
+        app = app.subcommand(
+            Command::new("uninstall")
+                .about("Uninstall the Phylum CLI")
+                .arg(arg!(
+                    -p --purge "Remove all files, including configuration files (default: false)"
+                )),
+        );
+    }
```

Comment:
I'm not sure a user that got the `phylum` executable without this *needs* to know? At least I'm puzzled as to why someone would run `phylum uninstall` after acquiring a version intentionally built without it.

Reply:
The situation I was thinking of is this:

* User installs `phylum` with a package manager
* Time passes. User forgets how they installed it.
* User wants to uninstall it, does a web search, and lands on `docs.phylum.io` that mentions `phylum uninstall`
* User runs `phylum uninstall`

But since we don't actually support any package managers, I'm not too worried about this. It's fine as is.